frontend/VolunteerOne/screens/Home/index.js

```diff
@@ -1,64 +1,58 @@
 import React from "react";
 import { StyleSheet, Dimensions, ScrollView } from "react-native";
 import { Block, theme } from "galio-framework";
-
-import { Button, Card } from "../../components";
-import articles from "../../constants/articles";
+import EventCard from "../../components/EventCard";
 const { width } = Dimensions.get("screen");
 
+const dataList = [
```

Comment:
I'm not sure if we want to have the dataList in the home page itself or in ```/constants``` (as ```events.js```) for readability purposes, but we can discuss